gitpod-init.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+cp .env_sample .env;
```

Comment:
**comment, non-blocking**: the semicolons are unneeded in the script - execution will proceed from top to bottom line by line automatically (I'm guessing you initially tried using separate statements in the init block in the yml file and this was a copy from an intermediate state).

Reply:
I'll remove them. Thanks @djuber!